en/deploy-tidb-binlog.md

```diff
@@ -396,6 +376,9 @@ The steps are as follows:
         ```shell
         kubectl exec binlogctl -n ${namespace} -- /binlogctl -pd-urls=https://${cluster_name}-pd:2379 -cmd update-pump -node-id ${cluster_name}-pump-${ordinal_id}:8250 --state offline -ssl-ca "/etc/binlog-tls/ca.crt" -ssl-cert "/etc/binlog-tls/tls.crt" -ssl-key "/etc/binlog-tls/tls.key"
         ```
+> **Note:**
+>
+> Do not scale in Pump nodes to 0. Otherwise, [Pump nodes are removed completely](#remove-pump-nodes-completely).
 
```

Comment:
Perhaps we could place this note in step 1 to avoid users setting ${pump_replicas} to 0.